notarize.js

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { notarize } = require('electron-notarize')
+
+exports.default = async function notarizing(context) {
+  const { electronPlatformName, appOutDir } = context
+
+  if (electronPlatformName !== 'darwin') {
+    return
+  }
+
+  if (process.env.SKIP_NOTARIZATION === 'true') {
+    console.log('SKIP_NOTARIZATION=true | Skipping the notarization"')
+    return
+  }
+
+  const appName = context.packager.appInfo.productFilename
+
+  return await notarize({
```

Comment:
remove `async` from function definition and return raw promise

Reply:
Removed "await". I'll keep "async" to make it clear that the function must return a promise.

Reply:
why does it need to return a promise?